src/main/resources/META-INF/spring/spring-mvc.xml

```diff
@@ -36,14 +36,12 @@
         <mvc:interceptor>
             <mvc:mapping path="/**" />
             <mvc:exclude-mapping path="/resources/**" />
-            <mvc:exclude-mapping path="/**/*.html" />
             <bean
                 class="org.terasoluna.gfw.web.logging.TraceLoggingInterceptor" />
         </mvc:interceptor>
         <mvc:interceptor>
             <mvc:mapping path="/**" />
             <mvc:exclude-mapping path="/resources/**" />
-            <mvc:exclude-mapping path="/**/*.html" />
             <bean
                 class="org.terasoluna.gfw.web.token.transaction.TransactionTokenInterceptor" />
         </mvc:interceptor>
```

Comment:
- [ ] Don't be required to change ?

https://github.com/terasolunaorg/terasoluna-gfw-web-blank/blob/ec49fdfc4d1dd8e852c8a059fee4f085d6cf1e96/src/main/resources/META-INF/spring/spring-mvc.xml#L51

https://github.com/terasolunaorg/terasoluna-gfw-web-blank/blob/ec49fdfc4d1dd8e852c8a059fee4f085d6cf1e96/src/main/resources/META-INF/spring/spring-mvc.xml#L60

Reply:
@bttakahashikkr Plz reply.

Reply:
@yoshikawaa
>Plz reply.

I'm sorry.

>Don't be required to change ?

I fixed it because I have the same opinon as your comment. 115b2029fe06009401c149fecd0d87d84f133f4e